cmake/configure.cmake

```diff
@@ -67,6 +67,12 @@ else()
     include_directories(${CUDA_TOOLKIT_INCLUDE})
 endif(NOT WITH_GPU)
 
+if(WITH_MKLDNN)
+    add_definitions(-DPADDLE_USE_MKLDNN)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
```

Comment:
I printed out the values of `${OpenMP_C_FLAGS}` and  `${OpenMP_CXX_FLAGS}`, and found there are `None`. Maybe `find_package(OpenMP REQUIRED)` is needed.